frontend/express/public/javascripts/countly/countly.template.js

```diff
@@ -1224,6 +1224,11 @@ var AppRouter = Backbone.Router.extend({
             path: [countlyGlobal.cdn + 'localization/min/'],
             mode: 'map',
             callback: function() {
+                if (countlyGlobal.company) {
+                    for (var key in jQuery.i18n.map) {
+                        jQuery.i18n.map[key] = jQuery.i18n.map[key].replace("Countly", countlyGlobal.company);
+                    }
+                }
```

Comment:
As far as I know, this will replace only first occurance of Countly, not others, if there are more than one Countly word in localized string